src/Unirest/Request.php

```diff
@@ -563,27 +594,27 @@ public static function send($method, $url, $body = null, $headers = array(), $us
 
         if (!empty(self::$auth['user'])) {
             curl_setopt_array(self::$handle, array(
-                CURLOPT_HTTPAUTH    => self::$auth['method'],
-                CURLOPT_USERPWD     => self::$auth['user'] . ':' . self::$auth['pass']
+                CURLOPT_HTTPAUTH => self::$auth['method'],
+                CURLOPT_USERPWD => self::$auth['user'] . ':' . self::$auth['pass']
             ));
         }
 
         if (self::$proxy['address'] !== false) {
             curl_setopt_array(self::$handle, array(
-                CURLOPT_PROXYTYPE       => self::$proxy['type'],
-                CURLOPT_PROXY           => self::$proxy['address'],
-                CURLOPT_PROXYPORT       => self::$proxy['port'],
+                CURLOPT_PROXYTYPE => self::$proxy['type'],
```

Comment:
We have kept these arrows `=>` below and above and below each other, to keep the code readable, kindly revert these changes